### ISSUE TITLE
Coloured text in TextBlocks

### ISSUE THREAD

**Issue by WalrusGumboot** [jwlodek/py_cui#79]
**Is your feature request related to a problem? Please describe.**
I'm writing a UI for a program I made that uses candlestick charts. For those who aren't familiar, a candlestick chart is a diagram used in finance to visualise what is happening to a certain company, they look like this: 
![image](https://user-images.githubusercontent.com/39743643/104136097-659d3980-5394-11eb-997b-692458feebb8.png)
The colour of the bars is important here: green means the price rose, red means it dropped. I'm using a library called [termgraph](https://github.com/sgeisler/termgraph) to draw them, and it in turn uses colorama to provide the colour.
**Describe the solution you'd like**
The ANSI escape codes (like `\u001b[31m` for red and `\u001b[32m` for green) should be rendered as that colour.

**Comment by jwlodek** [comment]
I agree that this would be a nice feature, but, as far as I know, curses (which is what `py_cui` uses under the hood for terminal rendering), does not support ANSI escape code based color markers. I think what would have to be done is to add functions that manually translate the escape codes into the supported format for the renderer, which would then convert those to curses commands to draw to the screen.

**Comment by robermp** [comment]
i'm also interested in this
